docs/tutorials/Bonding_and_Charge/mulliken_population.md

```diff
@@ -56,4 +56,130 @@ Si              2     1.356   2.644   0.000   0.000   4.000     0.000
+==========================================================================
+Ga              1     1.180   1.743   9.994   0.000  12.917     0.083
+As              1     1.488   3.595  10.000   0.000  15.083    -0.083
+==========================================================================
+
+            Bond                   Population      Length (A)
+======================================================================
+         Ga 1 -- As 1                   0.34        2.44652
+======================================================================
+```
+
+Despite having the same structure, the results are very different now:
+
+- Unlike before, the Ga and As ions have charges on them - this is indicative of ionic/polar character
+- There is now a much smaller population in the Ga-As bond. This indicates less covalent character
+- Especially in the case of Ga, the s:p shell ratio is far off from 1:3 - this again indicates that the bonding is not like in the cases above
+
+## Diatomic molecules
+
+Next we will examine a few diatomic molecules - HF, HCl and HBr.
+
+Here is the ```HF.cell``` file
+
+```
+%block lattice_abc
+5 5 5
+90 90 90
+%endblock lattice_abc
+
+%block positions_abs
+H 2 2 2
+F 2.91 2 2
+%endblock positions_abs
+```
+
+Since we are just trying to look at a single diatomic molecule, the cell is defined rather simply - an arbitrarily large cube for the cell (making it too large would make the calculation take longer, but make it too small and it won't be accurate), 1 atom placed about in the middle, and the 2^nd^ atom placed a bond length away from it - in this case F is $0.91 \mathring{A}$ to the right of the H . The bond lengths can be found on a [database](https://cccbdb.nist.gov/), or you may perform a [geometry optimisation](../../..//documentation/Geometry_Optimisation/overview) to find it yourself if you wish.
+
+Using a ```param``` file identical to before and running castep yields this towards the end of ```HF.castep```
+
+```
+Atomic Populations (Mulliken)
+-----------------------------
+Species          Ion     s       p       d       f      Total   Charge (e)
+==========================================================================
+H               1     0.309   0.000   0.000   0.000   0.309     0.691
+F               1     1.960   5.731   0.000   0.000   7.691    -0.691
+==========================================================================
+
+            Bond                   Population      Length (A)
+======================================================================
+          H 1 -- F 1                    0.34        0.91000
+======================================================================
+```
+There are a couple of interesting things to note:
+
+- Like GaAs (and unlike Si and diamond), there are 2 opposite charges on both atoms. However, the charge is significantly larger, indicating that the molecule is highly polar/ionic.
```

Comment:
This may be a personal thing and I'm not sure if chemists use it the same way, but I wouldn't describe a non-integer charge transfer as "ionic", although I know technically that is a thing that is used. To me "highly polar" is enough.

Reply:
Isn't part of what this is calculating the extent of ionic vs covalent character? That's kind of what I'm trying to say here. 

Also I'm a bit confused by if you can even call GaAs polar (did so a bit earlier - I left them all as polar/ionic since I'm not which to use or if both are appropriate) - it makes sense for a single molecule but can you do the same for a GaAs crystal where it'll all cancel out?

Reply:
> I wouldn't describe a non-integer charge transfer as "ionic", although I know technically that is a thing that is used. To me "highly polar" is enough.

I just did a test on NaCl to see what it looks like on an ionic crystal, and the charges are non-integer there as well (in fact they're actually smaller in NaCl than HF!). At what point would you call something ionic vs just polar?